src/FormBuilder/Routes/RegisterFormBuilderPageRoute.php

```diff
@@ -77,41 +80,32 @@ public function renderPage()
             wp_die(__('Donation form does not exist.'));
         }
 
-        $formBuilderStorage = (new EnqueueScript(
+        (new EnqueueScript(
             '@givewp/form-builder/storage',
             'src/FormBuilder/resources/js/storage.js',
             GIVE_NEXT_GEN_DIR,
             GIVE_NEXT_GEN_URL,
             'give'
-        ));
-
-        $formBuilderStorage
+        ))
             ->dependencies(['jquery'])
-            ->registerLocalizeData(
-                'storageData',
-                $formBuilderViewModel->storageData($donationFormId)
-            )
+            ->registerLocalizeData('storageData', $formBuilderViewModel->storageData($donationFormId))
             ->loadInFooter()
             ->enqueue();
 
-        wp_enqueue_script(
+        (new EnqueueScript(
             '@givewp/form-builder/script',
             $formBuilderViewModel->jsPathFromPluginRoot(),
-            $this->getRegisteredFormBuilderJsDependencies(
-                $formBuilderViewModel->jsDependencies()
-            ),
-            GIVE_NEXT_GEN_VERSION,
-            true
-        );
-
-        wp_localize_script('@givewp/form-builder/script', 'formBuilderData', [
-            'gateways' => $formBuilderViewModel->getGateways(),
-            'isRecurringEnabled' => defined('GIVE_RECURRING_VERSION') ? GIVE_RECURRING_VERSION : null,
-            'recurringAddonData' => [
-                'isInstalled' => defined('GIVE_RECURRING_VERSION'),
-            ],
-            'gatewaySettingsUrl' => admin_url('edit.php?post_type=give_forms&page=give-settings&tab=gateways'),
-        ]);
+            GIVE_NEXT_GEN_DIR,
+            GIVE_NEXT_GEN_URL,
+            'give'
+        ))
+            ->dependencies(
+                $this->getRegisteredFormBuilderJsDependencies(
+                    $formBuilderViewModel->jsDependencies()
+                )
+            )
+            ->loadInFooter()
+            ->enqueue();
```

Comment:
(from merge) switching to `wp_enqueue_script` was intentional here and should be reverted

Reply:
Ok - I'm following now. I'll get this documented for future us.

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/181/commits/f902db52916b8ca161c761e9450ac83c1aa29b09, https://github.com/impress-org/givewp-next-gen/pull/181/commits/7f34b404ba18ca1a51fba5f18b5c4655ac0fefb9

Reply:
@kjohnson thanks man!